Helpers/KnowledgeStoreHelper.cs

```diff
@@ -13,10 +13,10 @@ public class KnowledgeStoreHelper
 	{
 		private static CosmosClient cosmosClient = new CosmosClient(Environment.GetEnvironmentVariable("KS_Cosmos_Endpoint"), Environment.GetEnvironmentVariable("KS_Cosmos_Key"));
 
-        // The database we will create
+        // An existing Cosmos DB database for storing enriched output
         private static Database database = cosmosClient.GetDatabase(Environment.GetEnvironmentVariable("KS_Cosmos_DB"));
 
-        // The container we will create.
+        // An existing container to stored enriched output for each document
```

Comment:
nit pick:  typo here;  should read "An existing container to store enriched output for each document"